modules/nf-core/universc/mkgtf/main.nf

```diff
@@ -0,0 +1,40 @@
+process UNIVERSC_MKGTF {
```

Comment:
The `universesc mkgtf` actually uses `cellranger mkgtf` so it should be removed.